### ISSUE TITLE
Import fails without any further explanation when running out of space on SD Card

### ISSUE THREAD

**Issue by hssm** [ankidroid/Anki-Android#3339]
Originally reported on Google Code with ID 2443

```
* Import a deck with lots of media on the emulator with small SD card set.
* "Import failed" message pops up

SD card capacity low message should appear

```

Reported by `perceptualchaos2` on 2014-12-18 06:52:35

**Comment by mikehardy** [comment]
Cross linking with #3282

**Comment by mikehardy** [comment]
I just fixed this with #5790
